gapic/src/platform/linux/com/google/gapid/glcanvas/GlCanvas.java

```diff
@@ -48,7 +48,7 @@
   static {
     boolean loaded = false;
     try {
-      Library.loadSystem("linux_glcanvas");
+      Library.loadSystem("", "linux_glcanvas");
```

Comment:
I don't understand that change? The org.lwjgl.system.Library documentation does not seem to have a loadSystem() function that could take two strings?

https://javadoc.lwjgl.org/org/lwjgl/system/Library.html

Reply:
https://github.com/LWJGL/lwjgl3/blob/master/modules/lwjgl/core/src/main/java/org/lwjgl/system/Library.java#L61